src/components/ContextMenu/ContextMenu.jsx

```diff
@@ -350,12 +358,13 @@ const ContextMenu = createClass({
 
 		const targetElement = _.first(findTypes(this.props, ContextMenu.Target));
 		const targetChildren = _.get(targetElement, 'props.children', null);
+		const TargetElementType = targetElement.props.elementType;
 
 		const flyoutElement = _.first(findTypes(this.props, ContextMenu.FlyOut));
 		const flyProps = _.get(flyoutElement, 'props', {});
 
 		return (
-			<span ref='target' {...passThroughs} className={cx('&', className)} style={style}>
+			<TargetElementType ref='target' {...passThroughs} className={cx('&', className)} style={style}>
```

Comment:
this is an elegant hack, but we probably want a better solution to this at some point